google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/CloudApiManager.java

```diff
@@ -194,14 +411,31 @@ private static void notifyApiEnableSkipped(Set<CloudLibrary> apis, Project proje
     notification.notify(project);
   }
 
+  private static void notifyServiceAccountCreated(Project project, String name, Path downloadDir) {
+    Notification notification =
+        NOTIFICATION_GROUP.createNotification(
+            GctBundle.message("cloud.apis.service.account.created.title"),
+            null /*subtitle*/,
+            GctBundle.message("cloud.apis.service.account.created.message", name),
+            NotificationType.INFORMATION);
+    notification.notify(project);
+
+    ApplicationManager.getApplication()
+        .invokeLater(
+            () ->
+                Messages.showMessageDialog(
+                    GctBundle.message(
+                        "cloud.apis.service.account.key.downloaded.message", downloadDir),
+                    GctBundle.message("cloud.apis.service.account.key.downloaded.title"),
+                    GoogleCloudToolsIcons.CLOUD));
+  }
+
   private static String joinApiNames(Set<CloudLibrary> apis) {
     return apis.stream().map(CloudLibrary::getName).collect(Collectors.joining("<br>"));
   }
 
-  private static void setApiEnableProgress(
-      ProgressIndicator indicator, String apiName, String cloudProjectName, double fraction) {
-    indicator.setText(
-        GctBundle.message("cloud.apis.enable.progress.message", apiName, cloudProjectName));
+  private static void showProgress(ProgressIndicator indicator, String message, double fraction) {
```

Comment:
Nit: it feels like this method shows the progress (like it wasn't showing before), maybe `updateProgress()`?